bodhi/server/templates/user.html

```diff
@@ -91,7 +91,7 @@ <h3 class="pull-left nomargin nopadding">${user['name']}'s latest updates</h3>
         <a href="${urls['recent_updates_rss']}">
           RSS <span class="fa fa-rss"></span>
         </a>
-        <a href="${urls['recent_updates']}&status=testing">
+        <a href="${urls['recent_updates']}&status=testing&active_release=true">
```

Comment:
When I visit http://localhost:6543/updates/?user=bowlofeggs&status=testing&active_release=true (as linked from "Testing" at http://localhost:6543/users/bowlofeggs), I see an "Exclude old releases" button, but the active_release=true *should* mean that they are already excluded. I think the problem is that this should be ```active_releases``` instead of ```active_release```.